.bunnyshell/templates/showcase-container-patterns/README.md

```diff
@@ -0,0 +1,198 @@
+# Container Patterns Template Documentation
+
+## Overview
+The `container-patterns-tpl` template demonstrates how to utilize init and sidecar containers in Bunnyshell. It provides practical examples to illustrate their setup and interaction within a containerized environment.
+
+## Init Container
+The init container is responsible for executing preliminary tasks before the main application starts. In this example, the init container is seeding the database.
+
+### Init Container Definition
+To define an init container within the main container's configuration, you use the `init_containers` field. Here's the configuration snippet:
+
+```yaml
+# Within the definition of the main container bind the init container
+pod:
+  init_containers:
+    - from: init-db-seed
+      name: init-db-seed
+      shared_paths:
+        - path: /tmp
+          target:
+            path: /tmp
+            container: "@parent"
+          initial_contents: "@target"
+```
+
+### Init Container Implementation
+The init container uses the `busybox` image to create a file in the specified paths. Here is the definition of the init container component:
```

Comment:
- the init container is using the `postgres:latest` image
- I would add that this is a demo purpose seed, as we know the Postgres (and other DBs) docker images features, to mount a seed file under `/docker-entrypoint-initdb.d` (https://hub.docker.com/_/postgres -> Initialization scripts)
- and also emphasis that we attached the initContainer to the `backend` component, which is configured to depend on component `db`, and the seed script connects to the host named `db` which is the component name